js/views/buttonsView.js

```diff
@@ -191,7 +198,7 @@ export default class ButtonsView extends Backbone.View {
     this.checkResetSubmittedState();
     this.checkFeedbackState();
     this.onButtonStateChanged(null, this.model.get('_buttonState'));
-    this.onFeedbackMessageChanged(null, this.model.get('feedbackMessage'));
+    this.disableFeedbackButton();
```

Comment:
I think there are scenarios where this feedback could be enabled.
The extra feedbackTitle changed option does not need to be considered here as the passed attribute is forcing an enable, so this line of code can remain how it was, or possibly updated to:
`this.onFeedbackMessageChanged(null, true);`

```toggleFeedbackButton(enabled) {
    a11y.toggleEnabled(this.$('.js-btn-feedback'), enabled);
  }```

Reply:
Thanks, @cahirodoherty-learningpool !

So, on the master branch, "Show Feedback" is disabled when you first view the question. Once the question is answered or the attempts are used up, then "Show Feedback" is enabled (aside from the #265  issue).

> I think this refresh should always be forcing feedback as enabled. That seems to be the current behaviour anyway.

I don't believe we would always want the "Show Feedback" button to be enabled. Instead, what about enabling it after the question has been submitted? I modified your suggestion to include this, and it seems to work. There may be a specific use case that I'm not considering, though, so let me know what you think.

`this.onFeedbackMessageChanged(null, this.model.get('_isSubmitted'));`